### PR TITLE
Allow anyone to tag/untag/close issues.

### PR DESCRIPTION
**Pull request by Carreau**
This is (usually) reserved to people having commit right.
I'm hopping to foster participation by giving people the ability to
triage issue.

Mostly opening that for discussion.

### CODE REVIEW

**Comment by ivanov** [comment]
Democratizing tagging sounds good to me, not sure about closing... What about proposing to close? Although I suppose there's not much harm in closing issues, they can just be re-opened if need be. So I'm on board, not opposed.

**Review by ivanov — APPROVED** [review]

**Comment by ivanov** [comment]
Also, I think it would help to be explicit how someone can do this. Add instructions to `CONTRIBUTING.md` or some other discoverable place? Does a user without permissions just comment `@meeseeksdev[bot] tag windows`?

**Comment by willingc** [comment]
I absolutely think that triaging issues, labels, and even recommending closing (with sign-off by another person) would be welcome. Thanks @Carreau

**Review by willingc — APPROVED** [review]

**Comment by Carreau** [comment]
I really need to have meeseeks trigger only when it's name is at the
beginning of the line....

**Comment by Carreau** [comment]
Yes user w/o commit right can just mention the bot like you did, it will
only apply existing labels.
I believe if the user does not have the permission, it tells that to the
user.

If someone abuse the bot we can ban them from using the bot.

One question, which I believe we can tackle later is traceability if
someone mention the bot and delete their comment, we can't know who
triggered it without diving through the bot logs.

**Comment by Carreau** [comment]
> I absolutely think that triaging issues, labels, and even recommending closing (with sign-off by another person) would be welcome.

Having sign-off by or multiple +1 to close may not be super easy. I'll think about how to do it.

I've push some info in contributing.md on how to use the bot.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,3 +1,31 @@
+## Triaging Issues
+
+On the IPython repository we strive to trust users and give them responsibility,
+this is why than to one of our bot, any user can close issues, add and remove
+labels by mentioning the bot and asking it to do things on your behalf.
+
+To close and issue (or PR), even if it is not your, use the following:
```
Comment (willingc) [inline CONTRIBUTING.md]: ...even if you did not create it, use ...

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,3 +1,31 @@
+## Triaging Issues
+
+On the IPython repository we strive to trust users and give them responsibility,
```
Comment (willingc) [inline CONTRIBUTING.md]: responsibility. By using one of our bots, any contributor...

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1,3 +1,31 @@
...
+this is why than to one of our bot, any user can close issues, add and remove
+labels by mentioning the bot and asking it to do things on your behalf.
+
+To close and issue (or PR), even if it is not your, use the following:
+
+> @meeseeksdev close
+
+This command can be in the middle of another comments, but must start a line.
```
Comment (willingc) [inline CONTRIBUTING.md]: ...start on its own line.

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -1,3 +1,31 @@
...
+this is why than to one of our bot, any user can close issues, add and remove
+labels by mentioning the bot and asking it to do things on your behalf.
+
+To close and issue (or PR), even if it is not your, use the following:
+
+> @meeseeksdev close
+
+This command can be in the middle of another comments, but must start a line. 
+
+To add labels to an issue, as the bot to `tag` with a comma separated list of
```
Comment (willingc) [inline CONTRIBUTING.md]: ...issue, ask the...

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -1,3 +1,31 @@
...
+tags to add:
+
+> @meeseeksdev tag windows, documentation
+
+Only already pre-created tags can be added, and the list is so far limitted to `async/await`,
```
Comment (willingc) [inline CONTRIBUTING.md]: limited

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -1,3 +1,31 @@
...
+`backported`, `help wanted`, `documentation`, `notebook`, `tab-completion`, `windows`
+
+To remove a label, use the `untag` command:
+
+> @meeseeksdev untag windows, documentation
+
+The list of commands that the bot can do is larger and we'll be experimenting
```
Comment (willingc) [inline CONTRIBUTING.md]: We'll be adding additional capabilities for the bot and will share them here when they are ready to be used.

**Review by willingc — COMMENTED** [review]
Looks good @Carreau. A few edits and this is ready to merge.

**Comment by Carreau** [comment]
Thanks @willingc  I took your suggestions into account.

**Comment by Carreau** [comment]
Thanks all !

<!--
@meeseeksdev[bot] say Oweee looking forward to help !

--!>

**Pull request merged**